recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ test:
     - sksparse
     - sksparse.cholmod
   commands:
-    - nosetests -sv sksparse
+    - OPENBLAS_NUM_THREADS=1 nosetests -sv sksparse
```

Comment:
Running the tests with `OPENBLAS_NUM_THREADS=1` will still ship a broken package if the multi-threaded `openblas` is an issue. I remember a talk at SciPy mentioned that many bugs with multi-threaded openblas were considered "features" and would never be fixed :unamused: